modules/custom/az_demo/data/az_demo_block_content_block.json

```diff
@@ -10,7 +10,7 @@
           "settings": {
           "id" : "menu_block:main",
           "label" : "Main navigation",
-          "label_display" : false,
+          "label_display" : true,
```

Comment:
```suggestion
          "label_display" : "visible",
```

Although `true` seems to work in order for the title to be displayed, the "Display title" checkbox on the edit page for the custom block is not checked:

<img width="597" alt="image" src="https://user-images.githubusercontent.com/74572157/196293865-c64964a7-3ade-4a59-952e-e3d5ce716768.png">

From testing locally, changing `true` to `"visible"` resulted in the "Display title" checkbox being checked.